### PR TITLE
adding lesson01 python pushups

### PR DESCRIPTION
**Pull request by gnar5tar**
Completed break_me.py task and coding bat puzzles

### CODE REVIEW

File: students/zach_thomson/lesson02/fizzbuzz.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+def fizzBuzz():
```
Comment (natasha-aleksandrova) [inline students/zach_thomson/lesson02/fizzbuzz.py]: pep8 style violation: use snake_case style for var names and func names -> `fizz_buzz`

File: students/zach_thomson/lesson02/fizzbuzz.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    i = 1
+    while i <= 100:
```
Comment (natasha-aleksandrova) [inline students/zach_thomson/lesson02/fizzbuzz.py]: note that you could have used `range` here which is more appropriate

**Comment by natasha-aleksandrova** [comment]
don't forget to submit on the course website with a link to this PR

File: students/zach_thomson/lesson02/fizzbuzz.py (hunk 1 above)
Comment (gnar5tar) [reply students/zach_thomson/lesson02/fizzbuzz.py]: Ok, good to know. I will use that next time.

File: students/zach_thomson/lesson02/fizzbuzz.py (hunk 2 above)
Comment (gnar5tar) [reply students/zach_thomson/lesson02/fizzbuzz.py]: So it'd be more appropriate to have my while statement look like:

while i in range(101)

**Pull request merged**